src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -603,11 +602,9 @@ internal static ImmutableArray<TypedConstant> Encode(TypeSymbol type, TypeSymbol
                 return constantsBuilder.ToImmutableAndFree();
             }
 
-            internal static ImmutableArray<bool> Encode(TypeSymbol type, int customModifiersCount, RefKind refKind)
+            internal static void Encode(TypeSymbol type, int customModifiersCount, RefKind refKind, ArrayBuilder<bool> transformFlagsBuilder)
```

Comment:
Sorry, I meant: should the caller directly call the internal method (inline this call)?